wire-ios-sync-engine/Source/Synchronization/ZMOperationLoop+PushChannel.swift

```diff
@@ -31,7 +31,7 @@ extension ZMOperationLoop: ZMPushChannelConsumer {
             // fix it. Once we're sure it works, we should remove this.
             do {
                 let decoder = JSONDecoder.defaultDecoder
-                _ = try decoder.decode(UpdateEventEnvelope.self, from: data)
+                _ = try decoder.decode(UpdateEventEnvelopeV0.self, from: data)
```

Comment:
will this be enough in case of error, to know which event failed to parse and to fix the event parsing?

Reply:
yes, because internally there is a `UpdateEventDecodingProxy` that tells which event failed